src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperContent.cs

```diff
@@ -48,6 +49,25 @@ public abstract class TagHelperContent : IEnumerable<string>
         public abstract TagHelperContent Append(string value);
 
         /// <summary>
+        /// Appends formatted string by replacing each format item in the specified string 
+        /// with the text equivalent of a corresponding object's value.
+        /// </summary>
+        /// <param name="format">A composite format string.</param>
```

Comment:
Maybe link to the composite format string link: https://msdn.microsoft.com/en-us/library/txafckwd(v=vs.110).aspx